pkg/settings/registry.go

```diff
@@ -238,6 +238,7 @@ var retiredSettings = map[InternalKey]struct{}{
 	"kv.rangefeed.use_dedicated_connection_class.enabled":  {},
 	"sql.trace.session_eventlog.enabled":                   {},
 	"sql.show_ranges_deprecated_behavior.enabled":          {},
+	"physical_replication.enabled":                         {},
```

Comment:
I think we want the "key" here, which is cross_cluster_replication.enabled.

Reply:
Ugh, good catch. I don't know why we bothered with an alias for something that wasn't even officially released.